### PR TITLE
incorrect status bar fixes [CPP-927]

### PR DESCRIPTION
**Pull request by adrian-kong**
image

addresses RTK issue (conflicting variable in QML and js)

unsure if INS is fixed (might need replay file to verify)

### CODE REVIEW

File: console_backend/src/tabs/solution_tab/solution_position_tab.rs [hunk 1]
```diff
@@ -396,8 +398,6 @@ impl SolutionPositionTab {
         let dops_fields = msg.fields();
         self.table
             .insert(DOPS_FLAGS, format!("0x{:<03x}", dops_fields.flags));
-        self.table
-            .insert(INS_STATUS, format!("0x{:<01x}", self.ins_status_flags));
```
Comment (adrian-kong) [inline console_backend/src/tabs/solution_tab/solution_position_tab.rs:400]: not quite sure what this was doing in handle dops,
should INS status be updated based on DOPS updates?

File: console_backend/src/types.rs [hunk 2]
```diff
@@ -1108,8 +1108,8 @@ impl GnssModes {
             GnssModes::NoFix => NO_FIX_LABEL,
             GnssModes::Spp => SPP,
             GnssModes::Dgnss => DGNSS,
-            GnssModes::Float => RTK,
-            GnssModes::Fixed => RTK,
+            GnssModes::Float => FLOAT,
+            GnssModes::Fixed => FIXED,
```
Comment (adrian-kong) [inline console_backend/src/types.rs:1112]: this addresses the same 'RTK' label in status bar issue

File: console_backend/src/status_bar.rs [hunk 3]
```diff
@@ -274,6 +274,7 @@ impl StatusBar {
                 .heartbeat_data
                 .lock()
                 .expect(HEARTBEAT_LOCK_MUTEX_FAILURE);
+            shared_data.ins_status_string = format!("0x{:<01x}", msg.flags);
```
Comment (adrian-kong) [inline console_backend/src/status_bar.rs:277]: basically how solution table inserts ins status string, did we want to replicate the logic from the solution table?

File: console_backend/src/types.rs (hunk 2 above)
Comment (adrian-kong) [reply console_backend/src/types.rs]: moved to QML to fix issue

File: console_backend/src/status_bar.rs [hunk 4]
```diff
@@ -39,7 +39,7 @@ use crate::piksi_tools_constants::{
     ins_error_dict, ins_mode_dict, ins_type_dict, rtk_mode_dict, DR_MODE, EMPTY_STR, RTK_MODES,
 };
 use crate::shared_state::SharedState;
-use crate::types::{BaselineNED, GnssModes, PosLLH};
+use crate::types::{BaselineNED, Dops, GnssModes, PosLLH};
```
Comment (adrian-kong) [inline console_backend/src/status_bar.rs:42]:
```suggestion
use crate::types::{BaselineNED, GnssModes, PosLLH};
```

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4911083288)

**Comment by silverjam** [comment]
Do we understand why this broke?

**Comment by adrian-kong** [comment]
> Do we understand why this broke? Or did it ever work?
@silverjam 

think my PR https://github.com/swift-nav/swift-toolbox/pull/972 created this side effect.
variable naming conflicted with pyside and qml.
`set_rtk` in python class  and `rtk` in qml property which this changed the scope of variable assignments 😢 annoying bug thats super easy to overlook.
it might follow that we should consider naming qml properties uniquely to distinguish them from pyside variables in the future to prevent these types of weird behaviour

**Review by silverjam — APPROVED** [review]

File: console_backend/src/tabs/solution_tab/solution_position_tab.rs (hunk 1 above)
Comment (silverjam) [reply console_backend/src/tabs/solution_tab/solution_position_tab.rs:400]: @john-michaelburke any ideas?

@adrian-kong can look at the old console? Maybe some hints there.  I'm not sure why we'd handle this in DOPS either.

Comment (john-michaelburke) [reply console_backend/src/tabs/solution_tab/solution_position_tab.rs:400]: Yeah purely an artifact of copy pasta:
```
https://github.com/swift-nav/piksi_tools/blob/7725f247b978124f3ee7d23d67a0995f3aaa6f24/piksi_tools/console/solution_view.py#L573
```

**Pull request merged**